README.md

```diff
@@ -204,6 +204,23 @@ On Ubuntu 17 and above:
  wait for it to build.
  ```
 
+The above will create a working libcodec2.dll file for inclusion in other applications
+but the utility programs (e.g. `freedv-rx.exe`) are not guaranteed to work properly in 
+a native Windows environment. Usage of Codec2 via the Windows Subsystem For Linux (WSL) 
+is recommended:
```

Comment:
1. All good up to the end of the first sentence on line 209.
2. Beyond that I feel recommending WSL is going beyond (and indeed the opposite direction) to what we agreed at PLT - it implies ongoing support, and will lead to the same confusion that we are trying to resolve.  As per our PLT decision I'd prefer to leave it at the first sentence.
3. I'd also suggest we rm 

    ```
    Building for Windows on a Windows machine
    
    mkdir build_windows (Or what ever you want to call your build dir)
    cmake -G "MinGW Makefiles" -D CMAKE_MAKE_PROGRAM=mingw32-make.exe
    Or if you use ninja for building cmake -G "Ninja" ..
    mingw32-make or ninja  depends on what you used in the last command
    wait for it to build.
    ```
    as we aren't testing that regularly.

Reply:
Per my added comment below WSL is a Linux VM running inside Windows (with some support from the kernel to allow seamless access to file I/O etc.) so I'm not sure this is really supporting Windows per se. Though since it is basically still Linux, perhaps we don't need to have that paragraph either.